New Subsection for Newcomer Checks Tool in the Delegate Handbook (#558)
thewca/wca-documents: edudoc/delegate-handbook/delegate-handbook.md

- A localized name is permitted only for competitors with a legal name, as shown in a legal document, in a non-Roman script, or for competitors competing in a region where a localized name is necessary to identify them.
- While a localized name may contain multiple names, a competitor may not have multiple localized names.

### Newcomer Checks Tool

> The newcomer checks tool is a tab under the left navigation bar on competition webpages that serves to assist delegates with veryifying newcomers. It is intended to catch issues like duplicate competitor profiles, name format issues, or other new competitor data issues ahead of time to streamline check-in on competition days. To use it, first run the tool by clicking the run button. It will run the three separate sub-tools, the duplicate checker, the name format checker, and the date of birth checker, each given their own tab after it has finished.

Reviewer: ```suggestion
The Newcomer Checks tool is a tab under the left navigation bar on competition webpages that serves to assist delegates with verifying first-time competitors. It is intended to catch issues, including duplicate competitor profiles, name format issues, or other new competitor data issues, ahead of time to streamline check-in on competition days. To use it, run the tool by clicking the run button. It will run the three separate sub-tools: the duplicate checker, the name format checker, and the date of birth checker, each given its own tab after it has finished.
```

> - The duplicate checker checks if a competitor has created a new WCA account for themselves when they already have an existing account and WCA ID. It pulls the information (name, country, and date of birth) for each newcomer registered for a competition and compares it against existing competitiors. It prints the information for the closest existing competitiors to the newcomers at a competition and displays a name similarity score out of 100. This, along with comparing dates of birth, can indicate to a delegate if a newcomer already has a WCA ID. Data changes can be made directly within the interface by clicking the edit button of each competitor and making the necessary changes. If a competitor is confirmed to have made a duplicate account, the two profiles can be merged together in the interface using the merge button.

Reviewer: ```suggestion
- The Duplicate Checker checks whether a competitor has created a new WCA account when they already have an existing account and WCA ID. It pulls the information (i.e. name, country, and date of birth) for each newcomer registered for a competition and compares it against existing competitors. It prints the information for the closest existing competitors to the newcomers at a competition and displays a name similarity score out of 100. This, along with comparing dates of birth, can indicate to a Delegate if a newcomer already has a WCA ID. Data changes can be made directly within the interface by clicking the edit button of each competitor and making the necessary changes. If a competitor is confirmed to have made a duplicate account, the two profiles can be merged in the interface using the merge button.
```

> - The name format checker identifies any newcomer whose name does not conform to the name requirements or contains other inconsistencies in their name.

Reviewer: ```suggestion
- The Name Formats Checker identifies any newcomer whose name does not conform to the name requirements or contains other inconsistencies.
```

> - The date of birth checker identifies potential errors in a newcomer's date of birth (e.g. birth year set to the current year).

Reviewer: ```suggestion
- The Date of Birth (DOB) Checker identifies potential errors in a newcomer's date of birth (e.g. birth year set to the current year).
```

If inconsistencies arise, reach out to the competitor to ensure the correct information is being recorded.

> When using newcomer checks, make sure to re-synchronize with WCA Live after all changes have been made to ensure the updates are displayed across all other tools. Currently, newcomer checks are an optional tool to help identify data changes early and simplify the process of changing them. In the future, this tool will become mandatory before submitting results. It is important to note that using this tool does not replace newcomer verification at check-in of the competition. Newcomers should still be checked-in and verified day of.

Reviewer: ```suggestion
When using the Newcomer Checks feature, ensure you re-synchronize WCA Live after all the changes have been made to ensure the updates are displayed across all other tools. Currently, the Newcomer Checks feature is an optional tool to help identify data changes early and simplify the process of changing them. In the future, this tool will become a mandatory step to complete before submitting the results of the competition. It is important to note that using this tool does not replace newcomer verification at the check-in during the competition. First-time competitors should still be checked in and verified at the competition before they compete.
```
